core/store/migrate/migrations/0131_add_multi_user.sql

```diff
@@ -5,6 +5,10 @@ CREATE TYPE user_roles AS ENUM ('admin', 'edit', 'run', 'view');
 
 -- Add new role column to users table, type enum
 ALTER TABLE users ADD role user_roles NOT NULL DEFAULT 'view';
+
+-- We are migrating up from a single user full access user - this should be reflected as the admin 
+UPDATE users SET role = 'admin';
```

Comment:
How do we know there aren't any other, non-admin users in the DB when this is executed?

Reply:
Yeah I discussed this offline - at this point of migrating there would not be the multi user feature. This is an assumption we can make for migrate up as the general process should be to backup the DB before migrating and restore if necessary.

CC @samsondav @jmank88